lib/src/components/NxTransferList/__tests__/NxTransferList.test.tsx

```diff
@@ -22,7 +22,9 @@ describe('NxTransferList', function() {
         onSelectedItemsFilterChange: () => {},
         onChange: () => {}
       },
-      getShallow = getShallowComponent<Props<string | number>>(NxTransferList, minimalProps);
+      // eslint-disable-next-line no-spaced-func
+      getShallow = getShallowComponent<Props<string | number, Set<number | string> | (string | number)[]>>
```

Comment:
If I'm not mistaken this change is no longer valid

Reply:
> If I'm not mistaken this change is no longer valid

Yup! This was fixed here: c1f9f1f29542985e93f7345f1af4eaf152f3e952